### PR TITLE
Pipeline MRI Preview

### LINKED ISSUE #1502: Create a new pipeline for visualizing nodal value ((e.g. thickness/SUVR)) on brain surface
Create a new pipeline that produces a network with values in the diagonal, which produces 1 measure which is the same values in the diagonal.

### PR DESCRIPTION
**Pull request by egolol**

### CODE REVIEW

**Comment by giovannivolpe** [comment]
@egolol please, add also example file

File: braph2genesis/graphs/_GraphP.gen.m [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+graph = Graph.GRAPH;
+
+%%% ¡connectivity!
+connectivity = Graph.WEIGHTED;
+
+%%% ¡directionality!
+directionality = Graph.UNDIRECTED;
+
+%%% ¡selfconnectivity!
+selfconnectivity = Graph.NONSELFCONNECTED;
```
Comment (annacanal) [inline braph2genesis/graphs/_GraphP.gen.m]: It should be SELFCONNECTED ; if you want to have values in the diagonal

File: braph2genesis/pipelines/structural/_AnalyzeGroup_ST_Preview.gen.m [hunk 2]
```diff
@@ -0,0 +1,67 @@
+%% ¡header!
+AnalyzeGroup_ST_Preview < AnalyzeGroup (a, graph analysis with structural data) is a graph analysis using structural data.
+
+%% ¡description!
+This graph analysis uses structural data and analyzes them using weighted undirected graphs,
+binary undirected multigraphs with fixed thresholds,
+or binary undirected multigraphs with fixed densities.
```
Comment (annacanal) [inline braph2genesis/pipelines/structural/_AnalyzeGroup_ST_Preview.gen.m]: This graph analysis uses structural data and previews the raw data

**Review by annacanal — CHANGES_REQUESTED** [review]
the graph should be SELFCONNECTED; if you add an example file, you would detect this

**Pull request closed without merging**